packages/components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -51,11 +62,11 @@ export const TooltipRoot = ({ children, enterDelay = 700, open, onOpenChange }:
 TooltipRoot.displayName = 'Tooltip.Root';
 
 export const TooltipTrigger = (
-    { children, 'data-test-id': dataTestId = 'fondue-tooltip-trigger' }: TooltipTriggerProps,
+    { wrapChild = false, children, 'data-test-id': dataTestId = 'fondue-tooltip-trigger' }: TooltipTriggerProps,
     ref: ForwardedRef<HTMLButtonElement>,
 ) => {
     return (
-        <RadixTooltip.Trigger data-test-id={dataTestId} asChild ref={ref}>
+        <RadixTooltip.Trigger asChild={!wrapChild} data-test-id={dataTestId} ref={ref}>
```

Comment:
Why did we have to rename?

Reply:
Render delegation pattern usually use the `as` cc @noahwaldner

Reply:
Personally I'd go with `asChild`, I think it's as descriptive as `wrapChild` plus we didn't come up with it